lib/datadog/appsec/configuration.rb

```diff
@@ -40,6 +40,10 @@ def ip_denylist=(value)
           options[:ip_denylist] = value
         end
 
+        def user_id_denylist=(value)
+          options[:user_id_denylist] = value
```

Comment:
Wondering if we should rather resolve to a default here:

```suggestion
          options[:user_id_denylist] = value || []
```

Reply:
@lloeki I actually change the location in which we return the default empty array. Rather than on the setter I returned an empty array at the time of reading the value of `client_ip_denylist` or `user_id_denylist` if nothing has been configured. 

Check https://github.com/DataDog/dd-trace-rb/pull/2612/commits/2c89745ed0b7250b18f3a6013303b4b4b61891ec